src/lib/pageVideoStreamWriter.ts

```diff
@@ -167,6 +175,7 @@ export default class PageVideoStreamWriter extends EventEmitter {
       .videoCodec('libx264')
       .size(this.videoFrameSize)
       .aspect(this.options.aspectRatio || '4:3')
+      .autopad(this.autopad.activation, this.autopad.color)
```

Comment:
Since `autoPad.color` is optional, it would be better to use optional chaining here. 

`this.autopad?.color`

Reply:
GOTCHA!
Thank you for the advice.